codecov.yml

```diff
@@ -1,4 +1,10 @@
 coverage:
+  ignore:
+    - "test_*"
+    - "*/test_*"
+    - "*/*/test_*"
+    - "*/*/*/test_*"
+    - "*/*/*/*/test_*"
```

Comment:
I'm surprised there's no way to make this just two lines a la 
```
ignore:
  - "*/test_*"
```
maybe using full-featured regex instead of simple wildcards?